v2/api/containerservice/customizations/managed_cluster_extensions.go

```diff
@@ -142,7 +142,7 @@ func (ext *ManagedClusterExtension) PreReconcileCheck(
 	_ context.Context,
 	obj genruntime.MetaObject,
 	_ genruntime.MetaObject,
-	_ kubeclient.Client,
```

Comment:
Would we not require kubeClient for cluster operations?

Reply:
The resolver has a kubeClient already, I think there's a method to access it. If not, easy to add.